### DISCUSSION TITLE
Initial datasets for dpgen

### DISCUSSION THREAD

**Discussion by XiaMinqi** [deepmodeling/deepmd-kit#2209] [Q&A] [answered]
I'm going to use DPMD to investigate a water-graphite interface system, therefore I prepared structures of water, bulk carbon and water-carbon interface as training set. 
As the lattice parameters for these 3 kinds of structures are not the same, using the same k-points set may introduce some error for dp training, I suppose.
So in order to keep the results at the same level of precision, is it right to control the kspacing and ENCUT as the same for different structures in training set?

**Comment by njzjz** [comment, marked as answer]
Actually, different elements will not share the same neural networks.

**Reply by XiaMinqi** [reply]
I'm afraid I didn't comprehend your reply entirely. Do you mean there's no need to keep parameters for different structures in training set as the same value?

**Reply by njzjz** [reply]
You may need to keep the parameters the same, but this rule does not affect different atomic types. Of course, you don't need to make C and O the same.

**Reply by XiaMinqi** [reply]
Oh I see, thank you for your patience!
